IDV/verification.yaml

```diff
@@ -0,0 +1,864 @@
+openapi: 3.1.0
+info:
```

Comment:
In one of the meetings we agreed that the APi descriptions must be in Json format .

Reply:
@ingmarvali done. Moved them to JSON files.